scripts/compile-vrl-wasm.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -euo pipefail
+
+# wasm-compile-vrl.sh
+#
+# SUMMARY
+#
+#   Compiles relevant crates to wasm32-unknown-unknown
+
```

Comment:
I wonder if we should check only the `vrl/web-playground` crate, which should checked all the required crates for us, since it would need those as dependencies.

Reply:
One thing that comes to mind is the change that I was required to do to `lib/vrl/compiler/Cargo.toml` I had to add the js feature for getrandom to get it compiling to wasm32-unknown-unknown, which is the main goal of this task. 

I think it's good to have the crates separately because then it would prevent issues from other people trying to compile VRL to wasm32-unknown-unknown similar to what happened here:
https://github.com/gh123man/cgo-rust-perf-test/pull/3#discussion_r1012327985


Preventing the issue others might have https://github.com/gh123man/cgo-rust-perf-test/pull/3#discussion_r1012327985

Perhaps a document can be written up about how to compile VRL to WASM but not use the playground which would also prevent me from specifying the getrandom js feature in the `compiler/Cargo.toml`